css-positioning/relative.css

```diff
@@ -0,0 +1,39 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 50px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 80px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 150px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+}
```

Comment:
Instead of adding `position: relative` to each of your `id` selector rulesets, what ruleset do you have in your CSS file that would apply to all of the pokemon so that you only need to have `position: relative` once?